common-authentication/src/main/java/org/alfresco/hxi_connector/common/adapters/messaging/repository/AcsHealthProbe.java

```diff
@@ -89,13 +89,13 @@ void checkAcsAlive() throws InterruptedException
             }
         } while (timeout >= currentTime);
 
-        log.debug("ACS health probe failed after {} seconds. ACS is not available.", retryTimeoutSeconds);
+        log.info("ACS health probe failed after {} seconds. ACS is not available.", retryTimeoutSeconds);
```

Comment:
I think this could even be at `ERROR` since the application terminates afterwards. IMO it would be useful to include the URL `acsHealthEndpoint` too, as this would have helped us diagnose the missing configuration faster.

Reply:
Good point - changed to ERROR. 
Also changed the level on sleep part to WARN.

Reply:
Perfect :+1: